hologram_dashboard/server/request_handler/request_handler.cc

```diff
@@ -1,62 +1,69 @@
 /*
-    Copyright 2020 Google LLC
-
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        https://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
 */
+
 #include "request_handler.h"
+
 #include <vector>
 #include <sstream>
+
 using namespace Pistache;
+
 using namespace std;
 
+namespace wireless_android_play_analytics {
+
 RequestHandler::RequestHandler()
-    :server(Address(Ipv4::any(), Port(8000))) {
-        init();
+    :server_(Address(Ipv4::any(), Port(8000))) {
+        Init();
```

Comment:
nit: make the port a flag rather than hard-coded number.

Reply:
But according to my previous comment, I feel server related init could be in server.cc.

Reply:
As discussed offline, will resolve this in a separate pr.